tests/integration/test_db_isolation.py

```diff
@@ -81,25 +81,3 @@ def test_isolated_app_fixture_has_db_isolation_step2(pids_count_in_app, isolated
     # This proves that the step1 and the step2 are isolated.
     # The #PIDs must NOT have incremented.
     assert PersistentIdentifier.query.count() == pids_count_in_app
-
-
-def test_isolated_app_fixture_commit(isolated_app):
-    pids_count = PersistentIdentifier.query.count()
-
-    PersistentIdentifier.create(
-        pid_type='type1',
-        pid_value='value1',
-    )
-    db.session.commit()
-    assert PersistentIdentifier.query.count() == pids_count + 1
-
-
-def test_isolated_app_fixture_rollback(isolated_app):
-    pids_count = PersistentIdentifier.query.count()
-
-    PersistentIdentifier.create(
-        pid_type='type1',
-        pid_value='value1',
-    )
-    db.session.rollback()
-    assert PersistentIdentifier.query.count() == pids_count
```

Comment:
I think that both these tests are showing a wrong way of using `isolated_app`, so I deleted them: why would anyone want to commit or rollback explicitly in a test?

Reply:
What can happen is that you are testing code that commits/rolls back. Don't you want to test that those kind of tests work?

Reply:
I am fine with deleting these tests.
I believe we should replace them with tests that proves what happens when we should explicitly the `db.session.commit` and `rollback`. The current behavior is that there is no isolation, so prove that with a test.
in the long run, our isolation should work also in such cases.

And I believe we should be able to explicitly call commit and rollback in tests, just because we want to test explicit commit and rollback.